### ISSUE TITLE
Available for offline use as well, or just online?

### ISSUE THREAD

**Issue by dasabate** [MicrosoftDocs/powerapps-docs#3419]
Awesome feature! I'm testing this out with a customer on their Field Service Mobile App and have the impression that it is limited to online - i.e. not working for **offline profiles**, right?

Thank you!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c6ddb0f9-5f1e-021a-d12c-31500ef38ce7
* Version Independent ID: f3761398-40ab-b089-856a-3bdb81296869
* Content: [Command bar customization limitations - Power Apps](https://docs.microsoft.com/en-us/power-apps/maker/model-driven-apps/command-designer-limitations)
* Content Source: [powerapps-docs/maker/model-driven-apps/command-designer-limitations.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/maker/model-driven-apps/command-designer-limitations.md)
* Service: **powerapps**
* Sub-service: **mda-maker**
* GitHub Login: @caburk
* Microsoft Alias: **caburk**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @dasabate. This issue has been assigned to the doc author for review.

**Comment by caburk** [comment]
That's correct. Field Service does use some classic commands for offline, and that remains available on classic. We're working on better all-around offline capabilities with Power Fx, it's just not ready yet.
